management/server/http/users_handler.go

```diff
@@ -77,6 +83,7 @@ func (h *UsersHandler) UpdateUser(w http.ResponseWriter, r *http.Request) {
 		Role:       userRole,
 		AutoGroups: req.AutoGroups,
 		Blocked:    req.IsBlocked,
+		Issued:     existingUser.Issued,
```

Comment:
We want to keep `IntegrationReference` from existing user. The same is true for the group handler.